pg_audit_log.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'rails', '>= 4.2', '< 5.2'
+  spec.add_dependency 'rails', '>= 4.2', '< 5.3'
```

Comment:
Tiny little baby nit: since Rails 5.3 doesn't exist and probably won't ever exist, maybe this should be <= 5.2?

Reply:
I kinda agree with this in theory, although when trying it out i guess `5.2.4.3` is `>` than `5.2`.

Given that, would you be in favor of like `< `5.2.5` ? or `<= 5.2.4.3`?  
Having to be that specific makes me lean toward leaving as `< 5.3` tbh

Reply:
Ah, yeah, <5.3 makes sense then!